src/components/Feed/Socialize/CommentBox.tsx

```diff
@@ -0,0 +1,326 @@
+import styled, { css } from 'styled-components';
```

Comment:
this file captures this entire component 
<img width="419" alt="image" src="https://user-images.githubusercontent.com/6754223/195633340-cad19e01-d5a6-440e-81ce-ce93cc7d6d70.png">